detox/e2e/support/server_api/default_config.json

```diff
@@ -233,8 +233,8 @@
     },
     "NativeAppSettings": {
         "AppDownloadLink": "https://mattermost.com/download/#mattermostApps",
-        "AndroidAppDownloadLink": "https://about.mattermost.com/mattermost-android-app/",
-        "IosAppDownloadLink": "https://about.mattermost.com/mattermost-ios-app/"
+        "AndroidAppDownloadLink": "https://play.google.com/store/apps/details?id=com.mattermost.rn",
+        "IosAppDownloadLink": "https://apps.apple.com/us/app/mattermost/id1257222717"
```

Comment:
Why this change? I believe this is handled by a redirect?

Reply:
@enahum - Marketing is sunsetting a number of existing domains, and I'm working with Marketing to ensure that all links to the sunsetted domains are updated, including docs & product.

Yes, there are going to be redirects in place; however, we're formally updating all of the links so that we're no longer relying on those redirects being in place. There are a LOT of old links, like these two, across the product code.

Do you have a concern about updating these links from a development perspective? Or would changing these links cause downstream impact I haven't accounted for?

Reply:
no worries at all,.. just wanted to be sure